source/glossary.rst

```diff
@@ -144,6 +144,14 @@ Glossary
      A system :term:`topology` that uses more than one server and more than one drive per server to host a MinIO instance.
      MinIO recommends Kubernetes for distributed deployments.
 
+   multipart upload
+     Multipart upload is an :s3-docs:`Amazon S3 function <mpuoverview.html>` that splits a single object into multiple parts for moving from one location to another.
+     The separate parts upload indepently from one another and then MinIO reconstructs them into the original object.
+
+     MinIO recommends multipart upload for all objects larger than 100 MB.
```

Comment:
We might add *why* - we can borrow from the S3 docs here, but basically:

Multipart uploads provide benefits such as improved throughput and resiliency to network errors. Use multipart uploads for objects greater than 100MB in actual or estimated size for best results.

Reply:
Will address in #595